src/components/ui/Shell/Shell.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Shell } from "./Shell";
+
+export default {
+  title: "UI/Shell",
+  component: Shell,
+} as ComponentMeta<typeof Shell>;
+
+const Template: ComponentStory<typeof Shell> = (args) => {
```

Comment:
don't think we actually need variants on this story. Just dark mode selector should be good! Can remove args